src/Delegates/TasksQueryTrait.php

```diff
@@ -0,0 +1,138 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MeiliSearch\Delegates;
+
+trait TasksQueryTrait
+{
+    private int $next;
+    private array $types;
+    private array $statuses;
+    private array $indexUids;
+    private array $uids;
+    private array $canceledBy;
+    private \DateTime $beforeEnqueuedAt;
+    private \DateTime $afterEnqueuedAt;
+    private \DateTime $beforeStartedAt;
+    private \DateTime $afterStartedAt;
+    private \DateTime $beforeFinishedAt;
+    private \DateTime $afterFinishedAt;
+
+    public function setNext(int $next)
+    {
+        $this->next = $next;
+
+        return $this;
+    }
+
+    public function setTypes(array $types)
+    {
+        $this->types = $types;
+
+        return $this;
+    }
+
+    public function setStatuses(array $statuses)
+    {
+        $this->statuses = $statuses;
+
+        return $this;
+    }
+
+    public function setIndexUids(array $indexUids)
+    {
+        $this->indexUids = $indexUids;
+
+        return $this;
+    }
+
+    public function getIndexUids(): array
```

Comment:
Why is there only one getter?

Reply:
There is no need to retrieve this data from other places, that's why :)